apis/cloudfront/generator-config.yaml

```diff
@@ -1,3 +1,8 @@
+resources:
+  "Distribution":
+    api_versions:
+    - name: "v1alpha1"
```

Comment:
Configuration example for the `Distribution` resource. I've tested generating `Distribution` at `v1beta1`.